client/app/views/receiver/tip.html

```diff
@@ -79,19 +79,65 @@
   <div class="row">
     <div class="col-md-6">
       <div class="TipPageSectionTitle" data-translate>Submission</div>
-      <div class="col-md-12">
-        <dl>
-          <dt>
-            <span>{{::tip.context_name}}</span>
+
+      <div class="SubmissionInfoSection">
+      <table>
+
+        <tbody>
+        <tr>
+          <td class="left">
+            <span data-translate>Context</span>
+          </td>
+          <td class="right">
+            <span><b>{{::tip.context_name}}</b></span>
             <a data-ng-show="tip.context_description" popover="{{::tip.context_description}}" popover-placement="right" popover-trigger="mouseenter">
               <span class="glyphicon glyphicon-info-sign"></span>
             </a>
-          </dt>
-          <dt><span data-translate>Submission date</span></dt>
-          <dd>{{::tip.creation_date | date:'dd-MM-yyyy HH:mm'}}</dd>
-          <dt><span data-translate>Expiration Date</span></dt>
-          <dd>{{::tip.expiration_date | date:'dd-MM-yyyy HH:mm'}}</dd>
-        </dl>
+          </td>
+        </tr>
+        <tr>
+          <td class="left">
+            <span data-translate>Submission date</span>
+          </td>
+          <td class="right">
+            {{::tip.creation_date | date:'dd-MM-yyyy HH:mm'}}
+          </td>
+        </tr>
+        <tr>
+          <td class="left">
+            <span data-translate>Expiration Date</span>
+          </td>
+          <td class="right">
+            {{::tip.expiration_date | date:'dd-MM-yyyy HH:mm'}}
+          </td>
+        </tr>
+        <tr>
+          <td class="left">
+            <span data-translate>Whistleblower security</span>
+          </td>
+          <td class="right">
+            <span data-ng-hide="tip.tor2web"
+                  data-translate class="btn-success">Anonymous Submission</span>
+            <span data-ng-show="tip.tor2web"
+                  data-translate class="btn-warning">Confidential Submission</span>
+          </td>
+        </tr>
+
+
+
+        </tbody>
+      </table>
+      </div>
+
+      <hr />
+
+      <div class="TipPageSectionTitle" data-translate>The filled Questionnaire</div>
+      <small>
```

Comment:
let's try always to avoid the usage of small/b/i/and ather graphic modifier as it will make difficult for adopters to customize them with css; assign css classes and try to reuse them along the application.

http://www.w3.org/TR/html401/present/graphics.html

"Although they are not all deprecated, their use is discouraged in favor of style sheets."